CHANGELOG.rst

```diff
@@ -2,8 +2,17 @@ Changelog
 =========
 
 
-3.5.0
+3.5.1
 -----
+- Downgrade k8s Python SDK to >=11.0.0 (#14) [Yuan Tang]
+- Release 3.5 (#13) [binarycrayon]
```

Comment:
Change this to your first and last name to be consistent with above?

Reply:
I see. This is auto-generated. I think you'll want to add your name to your GitHub account so this will be consistent.

Reply:
ya I'll look into it, havn't had time to do that lately